src/cli.yaml

```diff
@@ -0,0 +1,27 @@
+name: Lotus GPU Miner
+args:
+  - config:
+      short: c
+      long: config
+      help: Configuration file
+      takes_value: true
+  - rpcaddress:
+      long: rpca-ddress
```

Comment:
Address is not good, as it sounds like a Bitcoin address is required.

Reply:
`rpc-host` would make more sense

Reply:
the code actually was using a different address anyways. Whoops.